boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts

```diff
@@ -90,6 +108,35 @@ arduino_serial: &lpuart2 {};
 	};
 };
 
+&is25wp064 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x000000000 0x00040000>;
+		};
+		slot0_partition: partition@40000 {
+			label = "image-0";
+			reg = <0x00040000 0x000150000>;
+		};
+		slot1_partition: partition@200000 {
```

Comment:
Why are there gaps in your map?  The region at `0x00190000` for `0x00700000` is not used.

Reply:
Also two of the register lengths are expressed as 40-bit integer literals which is confusing.  Remove one of the leading zeroes.

Reply:
Sorry. That part of the devicetree should probably not be part of the general devicetree for the EVK. I don't think other boards have MCUBOOT slots in their shipped files. I just included it here as an example use case of what I want to discuss later today during the dev meeting. I should have made that clear.